frontend/src/metabase/lib/formatting/numbers.tsx

```diff
@@ -180,11 +200,20 @@ export function formatChangeWithSign(
   return change > 0 ? `+${formattedNumber}` : formattedNumber;
 }
 
-export function numberFormatterForOptions(options: FormatNumberOptions) {
+export function numberFormatterForOptions(
+  options: FormatNumberOptions,
+): Intl.NumberFormat | null {
   options = {
     ...getDefaultNumberOptions(options),
     ...options,
   };
+
+  // datameasure is a custom format, not supported by Intl.NumberFormat
+  // Return null for datameasure - formatNumber will handle it specially
```

Comment:
There are a few places where comments or code still use `datameasure` instead of `datasize`

Reply:
Oops, will correct.